src/content/en/2020/markup.md

```diff
@@ -543,58 +608,63 @@ When we built our tests we didn’t check for the presence of images, but only l
 | … | | |
 | SVG | 5,328 | 0.08% |
 
-<figcaption>{{ figure_link(caption="Common favicon approaches.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Common favicon formats.", sheets_gid="1930085905", sql_file="pages_almanac_by_device_and_favicon_image_type.sql") }}</figcaption>
 </figure>
 
-There may be a couple of surprises in here:
+There are a couple of surprises in here:
 
-* Support for other formats is there but ICO is still the go-to format for favicons on the Web.
+* Support for other formats is there but ICO is still the go-to format for favicons on the web.
 * JPG is a relatively popular favicon format even though it may not yield the best results (or a comparatively large file size) for many favicon sizes.
```

Comment:
Please replace “file size” by “weight”—that avoids confusion with the other sizes (dimensions) we’re talking about.